requirements/prod.in

```diff
@@ -11,7 +11,7 @@ ansi2html
 bleach
 boto3
 channels
-channels-redis==4.0.0b1
+channels-redis @ https://github.com/django/channels_redis/archive/db5e8ff762862ea9bb274d9a12dfbd7f486ac8c5.tar.gz
```

Comment:
This fixes the [event loop issue](https://github.com/django/channels_redis/issues/312). With this fix I'm no longer missing websocket events (for example, Preflight completion events)